controller.js

```diff
@@ -31,8 +33,21 @@ const getArtById = (request, response, next) => {
         next(err)
     });
 };
+
+const getComByArt = (request, response, next) => {
+    const { article_id } = request.params;
+    Promise.all([fetchArtById(article_id), fetchComByArt(article_id)])
```

Comment:
I would look to extract the logic to the model

Reply:
Just to make sure I'm on the right track, by logic, do you mean the promise.all? (I wasn't sure as this is the way we did it in the lecture) so do you mean that I could invoke fetchArticleById in my fetchCommentByArticle function instead? Thank you!